confluent_server/confluent/config/configmanager.py

```diff
@@ -79,6 +79,7 @@
 import string
 import sys
 import threading
+import traceback
```

Comment:
No, it's solving a problem highlighted by IDE, that traceback was referenced even though not imported